test/controllers/recommendations.test.js

```diff
@@ -44,7 +44,7 @@ describe('Recommendations controller', () => {
   describe('user has no placement', () => {
     it('should return 404 Not Found', async () => {
       jest.spyOn(Recommendation, 'findById').mockResolvedValue(new Recommendation({id: 'abc123'}))
-      jest.spyOn(User, 'findOne').mockResolvedValue(new User({id: 123456, mcid: ['809xyz']}))
+      jest.spyOn(User, 'findAll').mockResolvedValue([new User({id: 123456, mcid: ['809xyz']})])
```

Comment:
Should these spies only return `id` now?

Reply:
sure